pages/instructorScanPaper/instructorScanPaper.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+
+const { processScrapPaperPdf } = require('../../lib/scrapPaperReader');
+const serverJobs = require('../../lib/server-jobs');
+
+const error = require('../../prairielib/lib/error');
+const ERR = require('async-stacktrace');
+
+router.get('/', (req, res) => {
+  res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
```

Comment:
As with the other PR, you'll need an authz check here too.

Reply:
Done.